src/psyclone/domain/lfric/kernel_interface.py

```diff
@@ -103,11 +118,12 @@ def generate(self, var_accesses=None):
             py:class:`psyclone.core.access_info.VariablesAccessInfo`
 
         '''
+        # pylint: disable=super-with-arguments
```

Comment:
My pylint (2.5.3) says "Bad option value 'super-with-arguments'".

Reply:
Removed. I guess you've got an older version of pylint.